lib/pages/add_bridge/error_message_dialog.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+// Display a showDialog with an error message related to the identifier
+void showErrorUsernameDialog(BuildContext context) {
```

Comment:
There's a lot of duplicate code in this file. I feel like some refactoring could simplify (and shorten) this file